ios/FluentUI.Demo/FluentUI.Demo/Demos/SideBarDemoController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+class SideBarDemoController: DemoController {
+	private var sideBar: SideBar?
```

Comment:
A minor that this is persistent here, convention is to sort swift files from most visible symbols to least visible so it roughly reads in the order of importance. So private vars generally belong towards the bottom https://microsoft.github.io/swift-guide/FileContentOrdering.html